src/Gameplay/WastePile.java

```diff
@@ -0,0 +1,44 @@
+package Gameplay;
+
+import Cards.Card;
+
+import java.awt.*;
+import java.lang.reflect.Array;
+import java.util.*;
+import java.util.List;
+
+public class WastePile extends Pile{
+
+    public WastePile(int x , int y){
+        super(x,y);
+        super.setSize(67, 97);
+        cards = new Stack<Card>();
```

Comment:
We can omit the generic type since it is specified in cards's declStmt *.
```suggestion
        cards = new Stack<>();
```

* declStmt = declaration statement.